Remove the not-very-useful "form" banner to save space.

http://b/206153000

--- a/Makefile.am
+++ b/Makefile.am
@@ -130,7 +130,6 @@
 	banners/confidential \
 	banners/secret \
 	banners/standard \
-	banners/form \
 	banners/topsecret \
 	banners/unclassified
 
@@ -282,10 +281,6 @@
 pkgcupsdatadir = $(CUPS_DATADIR)/data
 pkgcupsdata_DATA = \
 	data/default.pdf \
-	data/form_russian.pdf \
-	data/form_english.pdf \
-	data/form_english_in.odt \
-	data/form_russian_in.odt \
 	data/default-testpage.pdf \
 	data/testprint \
 	data/classified.pdf \
